File: Unidad 2-Estructuras de Control/ejercicio12.java
import java.util.Scanner;
public class Ejercico12
{
    public static void main(String[] args)
    {
        Scanner reader = new Scanner(System.in);                
int Numero1, Numero2;
                
    System.out.println("Ingrese El Primer Numero: ");
    Numero1 = reader.nextInt();

    System.out.println("Ingrese El Segundo Numero: ");
    Numero2 = reader.nextInt();


    if ( Numero1>Numero2) {

        System.out.printf (Numero1 + " " + Numero2);

    } else {

        System.out.printf (Numero2 + " " + Numero1);

}
    }
}

File: Unidad 2-Estructuras de Control/Ejercicio12.c
#include <stdio.h>
#include <math.h>
int main() 
{
    double num1, num2;
    printf("Ingrese el primer numero: ");
    scanf("%lf", &num1);

    printf("ingrese el segundo numero: ");
    scanf("%lf", &num2);

if ( num1>num2) {

    printf (" %lf, %lf. \n",num1, num2);

} else {
    
     printf (" %lf, %lf. \n",num2, num1);

}
    }
